pkg/apiserver/apiserver.go

```diff
@@ -117,14 +122,38 @@ func runE(flags *flags, log logr.Logger) error {
 	if err != nil {
 		return fmt.Errorf("creating client: %w", err)
 	}
+	dynamicClient, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		return fmt.Errorf("creating dynamic client: %w", err)
+	}
+
+	ctx := context.Background()
+	certPool := x509.NewCertPool()
+	certs, err := ioutil.ReadFile(flags.TLSCertFile)
+	if err != nil {
+		return err
+	}
+	certPool.AppendCertsFromPEM(certs)
+	grpcTLSConfig := &tls.Config{
```

Comment:
this can be inlined, no need for creating this variable.

Reply:
done